src/status.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-mutable-exports */
```

Comment:
- [ ] I would suggest fixing this in code instead of `disabling linters`. All rules are applied for the purpose to write code in a best practice manner.

Reply:
I fixed this issue